Added Title Scene and Loading Scene

Nemesis-AS opened the pull request:
Added a loading scene with a loading bar and added a title scene with a placeholder logo and text for buttons.

Nemesis-AS committed:
Added Title Scene and Loading Scene

thefinch commented:
Looks good so far. I'm working on some other stuff at the moment but I'll try to pull it down and test it tonight.

thefinch commented:
I pulled it down and tested it a bit. It looks like the hover area for the Save and Load buttons in the popup aren't quite right. Also we only need a load button so if you want we can skip the popup entirely or have multiple save slots and use it for that. Saving will be implement once the game is started, but I like that it already works.

Additional notes:
- Can we make it full screen?
- Let's make the buttons say "New Game" and "Load"

Nemesis-AS commented:
I think it would be better to replace the text with the buttons later when we get artists. And I will remove the popup and make it only the LOAD button. 
About the slots, we will need the popup to select the slot and it can be done easily too! 
And Phaser allows fullscreen mode but the graphics might suffer. So we have a choice there - Either we change the game canvas size to fullscreen or we scale the canvas.

Nemesis-AS committed:
Added Button Class, Load slots and fixed popup button bug

Nemesis-AS commented:
Things done in the new commit - 
- Added Button Class
- Added Load slots(thought they won't load any data as there is no saved data)
- Fixed Popup Button Bug
- Changed the names of the buttons

TO DO - 
- [ ] Make the "New Game" and "Load" buttons unclickable when the popup is open
- [ ] Add Fullscreen mode
